jobs/gcp-discovery/src/main/java/com/tmobile/pacbot/gcp/inventory/collector/GKEClusterInventoryCollector.java

```diff
@@ -73,16 +72,10 @@ public List<GKEClusterVH> fetchGKEClusterInventory(ProjectVH project) throws IOE
                     gkeClusterVH.setProjectName(project.getProjectName());
                     gkeClusterVH.setProjectId(project.getProjectId());
                     gkeClusterVH.set_cloudType(InventoryConstants.CLOUD_TYPE_GCP);
-
+                    gkeClusterVH.setVersion(cluster.getCurrentMasterVersion());
                     gkeClusterVH.setRegion(cluster.getLocation());
                     gkeClusterVH.setRegion(cluster.getLocation());
-                    if (cluster.getMasterAuthorizedNetworksConfig() != null) {
-                        HashMap<String, Object> masterAuthorizedNetworksConfigMap = new Gson().fromJson(
-                                cluster.getMasterAuthorizedNetworksConfig().toString(),
-                                HashMap.class);
 
-                        gkeClusterVH.setMasterAuthorizedNetworksConfig(masterAuthorizedNetworksConfigMap);
-                    }
```

Comment:
Please confirm once if these deletion is expected.

Reply:
as discussed with @AnjaliMadhavi-Nakirikanti  the code to set masterAuthorizedNetworksConfigMap was incorrect due to which GKEClusterInventoryCollector was throwing an error. Therefore, I have removed the code as of now to test this policy. However, @AnjaliMadhavi-Nakirikanti is working on rectifying code for masterAuthorizedNetworksConfigMap